liberty-maven-plugin/src/it/deploy-loose-config-apps-it/src/test/java/net/wasdev/wlp/test/servlet/it/KeystorePasswordTest.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
```

Comment:
I think you probably copied this over from another test. Don't think it is needed for this test case.

Reply:
Removed from this project and deploy-loose-config-it.